jdk/test/sun/security/krb5/auto/ReplayCacheTestProc.java

```diff
@@ -23,11 +23,11 @@
 
 /*
  * @test
- * @bug 7152176
+ * @bug 7152176 8168518
  * @summary More krb5 tests
  * @library ../../../../java/security/testlibrary/
```

Comment:
The library path `/test/lib` looks missing.

Reply:
I have made a PR for this exact problem: https://github.com/openjdk/jdk8u-dev/pull/478